applications/GeoMechanicsApplication/tests/test_triangle_15n.gid/test_triangle_15n.mdpa

```diff
@@ -0,0 +1,279 @@
+  17 1.0000000000 1.0000000000 0.0000000000
+  18 0.8750000000 1.0000000000 0.0000000000
+  19 0.7500000000 1.0000000000 0.0000000000
+  20 0.6250000000 1.0000000000 0.0000000000
+  21 0.5000000000 1.0000000000 0.0000000000
+  22 0.3750000000 1.0000000000 0.0000000000
+  23 0.2500000000 1.0000000000 0.0000000000
+  24 0.1250000000 1.0000000000 0.0000000000
+  25 0.0000000000 1.0000000000 0.0000000000
+  26 0.0000000000 0.8750000000 0.0000000000
+  27 0.0000000000 0.7500000000 0.0000000000
+  28 0.0000000000 0.6250000000 0.0000000000
+  29 0.0000000000 0.5000000000 0.0000000000
+  30 0.0000000000 0.3750000000 0.0000000000
+  31 0.0000000000 0.2500000000 0.0000000000
+  32 0.0000000000 0.1250000000 0.0000000000
+  33 0.5000000000 0.5000000000 0.0000000000
+  34 0.1250000000 0.5000000000 0.0000000000 
+  35 0.2500000000 0.5000000000 0.0000000000
+  36 0.3750000000 0.5000000000 0.0000000000
+  37 0.6250000000 0.5000000000 0.0000000000
+  38 0.7500000000 0.5000000000 0.0000000000
+  39 0.8750000000 0.5000000000 0.0000000000
+  40 0.5000000000 0.1250000000 0.0000000000
+  41 0.5000000000 0.2500000000 0.0000000000
+  42 0.5000000000 0.3750000000 0.0000000000
+  43 0.5000000000 0.6250000000 0.0000000000
+  44 0.5000000000 0.7500000000 0.0000000000
+  45 0.5000000000 0.8750000000 0.0000000000
+  46 0.6250000000 0.1250000000 0.0000000000
+  47 0.7500000000 0.2500000000 0.0000000000
+  48 0.8750000000 0.3750000000 0.0000000000
+  49 0.1250000000 0.1250000000 0.0000000000
+  50 0.2500000000 0.2500000000 0.0000000000
+  51 0.3750000000 0.3750000000 0.0000000000
+  52 0.6250000000 0.6250000000 0.0000000000
+  53 0.7500000000 0.7500000000 0.0000000000
+  54 0.8750000000 0.8750000000 0.0000000000
+  55 0.1250000000 0.6250000000 0.0000000000
+  56 0.2500000000 0.7500000000 0.0000000000
+  57 0.3750000000 0.8750000000 0.0000000000
+  58 0.2500000000 0.1250000000 0.0000000000
+  59 0.3750000000 0.1250000000 0.0000000000
+  60 0.7500000000 0.1250000000 0.0000000000
+  61 0.8750000000 0.1250000000 0.0000000000
+  62 0.1250000000 0.2500000000 0.0000000000
+  63 0.3750000000 0.2500000000 0.0000000000
+  64 0.6250000000 0.2500000000 0.0000000000
+  65 0.8750000000 0.2500000000 0.0000000000
+  66 0.1250000000 0.3750000000 0.0000000000
+  67 0.2500000000 0.3750000000 0.0000000000
+  68 0.6250000000 0.3750000000 0.0000000000
+  69 0.7500000000 0.3750000000 0.0000000000
+  70 0.2500000000 0.6250000000 0.0000000000
+  71 0.3750000000 0.6250000000 0.0000000000
+  72 0.7500000000 0.6250000000 0.0000000000
+  73 0.8750000000 0.6250000000 0.0000000000
+  74 0.1250000000 0.7500000000 0.0000000000
+  75 0.3750000000 0.7500000000 0.0000000000
+  76 0.6250000000 0.7500000000 0.0000000000
+  77 0.8750000000 0.7500000000 0.0000000000
+  78 0.1250000000 0.8750000000 0.0000000000
+  79 0.2500000000 0.8750000000 0.0000000000
+  80 0.6250000000 0.8750000000 0.0000000000
+  81 0.7500000000 0.8750000000 0.0000000000
+End Nodes
+
+
+Begin Elements SmallStrainUPwDiffOrderElement2D15N
```

Comment:
can you reduce number of elements to reduce the size of the mdpa file?

Reply:
Reduced the number of elements from 8 to 4.